### PR TITLE
fixed names of the classes: either to be passed or generated inside

### PR DESCRIPTION
**Pull request by lutich**
Please, consider taking part of the training set for validation purposes within `images_from_array` to keep it in sync with other `images_from_*`.

Note if no validation set is used the `preproc=None`.

### CODE REVIEW

**Comment by amaiya** [comment]
This change has been merged and included in v0.14.6.  In addition. the `images_from_array` function will now split off part of training as validation if no validation data is supplied, as you suggested.  A couple other issues with `images_from_array` were also fixed, as well.  See [this example notebook](https://github.com/amaiya/ktrain/blob/master/examples/vision/mnist-images_from_array_example.ipynb) demonstrating some of your contributions.  Thanks again!

**Comment by lutich** [comment]
Many thanks!
